lte/gateway/deploy/roles/agw_docker/files/magma_ifaces_gtp

```diff
@@ -0,0 +1,77 @@
+auto gtp_br0
```

Comment:
@ymasmoudi and I talked about this the other day in our review. The reason the code is repeated here is because some tasks on the baremetal need to be skipped and a decision was made to leave the current working code as-is while the feature is alpha. Before going to beta, a refactor is planned which would unify these streams where some tasks can be skipped for the containerized AGW.